--- a/lua/nvchad/au.lua
+++ b/lua/nvchad/au.lua
@@ -31,22 +31,6 @@
   })
 end
 
--- reload the plugin!
-autocmd("BufWritePost", {
-  pattern = vim.tbl_map(function(path)
-    return vim.fs.normalize(vim.uv.fs_realpath(path))
-  end, vim.fn.glob(vim.fn.stdpath "config" .. "/lua/**/*.lua", true, true, true)),
-  group = vim.api.nvim_create_augroup("ReloadNvChad", {}),
-
-  callback = function(opts)
-    local fp = vim.fn.fnamemodify(vim.fs.normalize(vim.api.nvim_buf_get_name(opts.buf)), ":r") --[[@as string]]
-    local app_name = vim.env.NVIM_APPNAME and vim.env.NVIM_APPNAME or "nvim"
-    local module = string.gsub(fp, "^.*/" .. app_name .. "/lua/", ""):gsub("/", ".")
-
-    require("nvchad.utils").reload(module)
-    -- vim.cmd("redraw!")
-  end,
-})
 
 vim.api.nvim_create_user_command("MasonInstallAll", function()
   require("nvchad.mason").install_all()
